rules/integrations/endpoint/elastic_endpoint_security_ransomware_detected.toml

```diff
@@ -0,0 +1,75 @@
+[metadata]
+creation_date = "2024/03/24"
+integration = ["endpoint"]
+maturity = "production"
+min_stack_comments = "New fields added: required_fields, related_integrations, setup"
+min_stack_version = "8.3.0"
+updated_date = "2024/03/24"
+promotion = true
+
+[rule]
+author = ["Elastic"]
+description = """
+Generates a detection alert each time an Elastic Endpoint Security alert for ransomware are received. Enabling this rule allows you to
+immediately begin investigating your Endpoint ransomware alerts. This rule identifies Endpoint Security ransomware detections only, and does not include prevention alerts.
+"""
+enabled = true
+from = "now-10m"
+index = ["logs-endpoint.alerts-*"]
+language = "kuery"
+license = "Elastic License v2"
+max_signals = 10000
+name = "Ransomware - Detected - Endpoint Security"
+references = [
+    "https://github.com/elastic/protections-artifacts/tree/main/ransomware",
+    "https://docs.elastic.co/en/integrations/endpoint"
+]
+risk_score = 47
+rule_id = "0c74cd7e-ea35-11ee-a417-f661ea17fbce"
+rule_name_override = "message"
+severity = "medium"
```

Comment:
imo risk score for Ransomware, shellcode and memory signature should default to high